data/tips-data.js

```diff
@@ -0,0 +1,32 @@
+module.exports = [
+  {
+    tip: 'Take a moment and drink some water. Consider adding lemon or cucumber for a little flavor.'
+  },
+  {
+    tip: 'Pause from your work and take a few deep breaths. Inhale through your nose for five seconds. Exhale through your mouth for five seconds. Repeat five times.'
+  },
+  {
+    tip: 'Give your eyes a break. Find an object that is about twenty feet away and focus on it for twenty seconds.'
+  },
+  {
+    tip: 'Let\'s take a break. Step away from your computer (and any other screens) for 5-10 minutes.'
+  },
+  {
+    tip: 'Time for a break. Do nothing for the next two minutes. (See http://www.donothingfor2minutes.com/ for guidance) '
+  },
+  {
+    tip: 'You\'ve been working hard! Take a lap around your office or home.'
+  },
+  {
+    tip: 'Take out a notepad and pen and write for a few minutes. Don’t worry if nothing brilliant comes to mind — even just writing down your feelings or what you\'re grateful for can be incredibly relaxing.'
+  },
+  {
+    tip: 'Close your eyes and take ten deep breaths.'
+  },
+  {
+    tip: 'Music break! Listen to one track that relaxes or energizes you. Dance it out if the feeling strikes you! [Consider headphones if you have coworkers nearby.]'
```

Comment:
Awesome!

Reply:
These are rad! I really like the level of detail provided.